bridges/TelegramBridge.php

```diff
@@ -249,7 +249,11 @@ private function processPhoto($messageDiv) {
 	}
 
 	private function processDate($messageDiv) {
-		return $messageDiv->find('time', 0)->datetime;
+		if ($messageDiv->find('time', 0)->datetime) {
```

Comment:
Instead of using an if statement, you could just use the `time` from the `span.tgme_widget_message_meta`.
```php
$messageMeta = $messageDiv->find('span.tgme_widget_message_meta', 0);
return $messageMeta->find('time', 0)->datetime;
```

Reply:
Yes, that's neater, I've committed your idea. Thanks for the bridge. 👍